packages/components/src/ProgressBar/ProgressBar.css

```diff
@@ -20,6 +27,10 @@ progress.ProgressBar[value]::-webkit-progress-bar {
   background-color: var(--color-surface--background);
 }
 
+progress.SteppedProgressBar[value]::-webkit-progress-bar {
+  background-color: var(--color-greyBlue--lightest);
```

Comment:
Both variations should use the same background color, so we don't need this extra declaration

Reply:
For context, we had adjusted the inactive state background colour to be a bit darker than what Atlantis version had because it was a bit harder to see in the stepped variation. If we want to maintain consistency between the two variations I can change this back!

`var(--color-surface--background)`:
<img width="920" alt="image" src="https://github.com/GetJobber/atlantis/assets/82672426/30a90a29-26bf-451b-a01b-0a2dd7f0759c">

`var(--color-greyBlue--lightest)`:
<img width="920" alt="image" src="https://github.com/GetJobber/atlantis/assets/82672426/c3a98d9c-6997-4426-8828-65e3ef400f7a">

Reply:
Fair! There is a PR against our [design package](https://github.com/GetJobber/atlantis/pull/1545) that will update `surface--background` to map to `greyBlue--lightest` in the next few-ish days so let's use the existing design `surface--background` token and let it absorb the updates when that lands.

Reply:
Oh good to know! Sounds good :+1: Fixed in [`cb7b493`](https://github.com/GetJobber/atlantis/pull/1653/commits/cb7b493612510ae26de4f8e87bba85472c8c6e3b).